task/git-cli/0.2/README.md

```diff
@@ -0,0 +1,60 @@
+# Git Task
+
+This `Task` is Git task to work with repositories used by other tasks
+in your Pipeline.
+
+## `git-cli`
+
+This [task](../0.2/git-cli.yaml) can be used to perform `git operations`.
+All git commands can be found [here](https://git-scm.com/docs).
+
+Command that needs to be run can be passed as a script to the task.
+
+### Workspaces
+
+* **source**: A workspace that contains the fetched git repository.
+* **input**: A workspace that contains file that needs to be added to git.
+* **ssh-directory**: An optional workspace to provide SSH credentials. At
+  minimum this should include a private key but can also include other common
+  files from `.ssh` including `config` and `known_hosts`. It is **strongly**
+  recommended that this workspace be bound to a Kubernetes `Secret`.
+* **basic-auth**: An optional workspace containing `.gitconfig` and
+  `.git-credentials` files. This allows username/password/access token to be
+  provided for basic auth.
+
+  It is **strongly** recommended that this workspace be bound to a Kubernetes
+  `Secret`. For details on the correct format of the files in this Workspace
+  see the `git-clone` (0.4) documentation as this task is based on this.
```

Comment:
Suggest copying the relevant sections over to this doc.  Probably this: https://github.com/tektoncd/catalog/tree/main/task/git-clone/0.4#using-basic-auth-credentials ?

Reply:
Copied!